invenio_app_ils/config.py

```diff
@@ -1276,3 +1339,6 @@ def _(x):
 }
 
 OPENDEFINITION_JSONRESOLVER_HOST = "inveniosoftware.org"
+
+FILES_REST_PERMISSION_FACTORY = "invenio_app_ils.permissions:files_permission"
+ILS_FILES_PATH = '/tmp/ils-files'
```

Comment:
I would not have this configurable, because it is only used once and that's it. Would it be possible to have a default location as in Invenio, current venv, and eventually pass it as param in the CLI?